package.json

```diff
@@ -19,6 +19,8 @@
   ],
   "license": "BSD-3-Clause",
   "dependencies": {
+    "@whatwg-node/fetch": "0.5.1",
```

Comment:
Why is this dependency required? We are already using a fetch dependency.

Reply:
It is required to limit multipart requests in v3;
https://the-guild.dev/graphql/yoga-server/docs/migration/migration-from-yoga-v2#no-more-node-specific-multipart-configuration
It is not an extra dependency actually. It is already dependency of Yoga.

Reply:
> Why do we need to add it as a Parse Server dependency then?
It will be an implicit dependency if I don't add it. I can remove it if you are fine with implicit dependencies.

Reply:
Yoga doesn't handle file size limits and so on because those are not part of Fetch API.
`createFetch` generates a Fetch API implementation here by limiting `Request.formData`.